content/blog/2023-06-21-comparison-of-ethereum-classic-and-fiat-monetary-policies/index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Comparison of Ethereum Classic and Fiat Monetary Policies"
+date: 2023-06-21
+author: Donald McIntyre
+contributors: ["DonaldMcIntyre"]
+tags: ["education"]
+linkImage: ./banner.png
+---
+
+---
+**You can listen to or watch this video here:**
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/aoY6XT6MHM4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+---
+
+![ETC vs fiat currencies.](./banner.png)
+
+In this post we will make a comparison of the Ethereum Classic (ETC) and fiat monetary policies.
+
+We will use ETC as base money and fiat money as base money. This is, that we will describe ETC’s issuance and stock at the network protocol layer without taking into account any fractional reserve multiplier that may occur at the exchanges and dapps that use crypto for margin lending and lending in general.
+
+In the same way, we will use the classical policies pertaining fiat money with regards to what is called “base money” or the money that the central bank issues without taking into account the multiplier of the money that occurs in the fractional reserve system that is managed by the rest of the banks of the economy.
+
+When referring to “fiat money” generically, we will mainly describe the way the United States Federal Reserve manages the base money. However, this is nearly identical in all monetary systems in all national currencies across the world.
+
+We start with the comparison table below and then we will explain each line.
+
+![Comparison between ETC and fiat monetary policies.](./1.png)
+
+## Digital Gold
+
+ETC is digital gold and fiat money is not.
+
+ETC is digital gold because it is a proof of work blockchain and the purpose of that consensus mechanism is to make it as costly to create ETC as it is costly to produce gold in the real world.
+
+The way it works is that miners in the blockchain create blocks of transactions, and to stamp them cryptographically they need to do a lot of computational work, which uses a lot of electricity. Then, when the blocks are accepted by the rest of the network, the miners are credited newly issued coins by the algorithm.
+
+The above system guarantees that ETC will be sound money as gold was in the old days, hence the term “digital gold”.
+
+Fiat money is not gold nor digital gold because a) it ended the gold standard decades ago and b) there is hardly any cost whatsoever to print more fiat currency.
```

Comment:
Line 41:  Rewrite for accuracy;
```suggestion
Fiat money is not gold nor digital gold because a) the gold standard ended decades ago and b) there is hardly any cost whatsoever to print more fiat currency.
```